server/src/main/java/org/opensearch/common/blobstore/BlobContainer.java

```diff
@@ -191,4 +195,48 @@ default long readBlobPreferredLength() {
      * @throws  IOException if there were any failures in reading from the blob container.
      */
     Map<String, BlobMetadata> listBlobsByPrefix(String blobNamePrefix) throws IOException;
+
+    /**
+     * The type representing sort order of blob names
+     */
+    enum BlobNameSortOrder {
```

Comment:
given most of the object store provide only `LEXICOGRAPHIC` , should we just have that only one option ? This would provide motivation to name keys as per that . If a user needs reverse, they can also do sort by themselves.

Reply:
Are you suggesting `listBlobsByPrefixInSortedOrder` to always return in lexicographic order and remove `BlobNameSortOrder` or BlobNameSortOrder to have only one value: `LEXICOGRAPHIC`?